src/FsAutoComplete.Core/CompilerServiceInterface.fs

```diff
@@ -365,7 +365,7 @@ type FSharpCompilerServiceChecker() =
   }
 
   member __.GetDeclarations (fileName, source, options, version) = async {
-    let! parseResult = checker.ParseFileInProject(fileName, source, options)
+    let! parseResult = checker.ParseFile(fileName, source, options)
```

Comment:
I'm surprised there is only one call to this.  Does FsAutoCOmplete provide breakpoint validation?  That should use ParseFile only (and will be faster after this update)

Reply:
Nope, we don't do breakpoint validation. Only editor with integrated debugger is Code, and we don't own debugger there - it's part of C# Omnisharp extension, and we have no way to plug custom behavior to it. Also I'm not sure to what extent breakpoint validation would be possible anyway given Code extensions API.

`GetDeclarations` is used by couple of features- go-to symbol in file, CodeLenses (finding positions where they should be displayed) and new Code Outline panel.